org.freedesktop.Sdk.Extension.haskell.json

```diff
@@ -0,0 +1,96 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.haskell",
```

Comment:
Wouldn't `ghc` a better name than `haskell`?

Reply:
I'd rather name it `haskell-platform`, since that includes cabal and stack as well.
Or should we split it up into something like `ghc` and `haskell-plaform`?